app/iot/devices.py

```diff
@@ -6,59 +7,76 @@
 TIME_TO_SLEEP = 0.5
 
 
-# of course this code looks dumb, but imagine some real implementations of each method here
+# of course this code looks dumb,
+# but imagine some real implementations of
+# each method here
 class HueLightDevice:
-    def connect(self) -> None:
+    async def connect(self) -> None:
         print("Connecting Hue Light.")
         time.sleep(TIME_TO_SLEEP)
         print("Hue Light connected.")
 
-    def disconnect(self) -> None:
+    async def disconnect(self) -> None:
         print("Disconnecting Hue Light.")
         time.sleep(TIME_TO_SLEEP)
         print("Hue Light disconnected.")
 
-    def send_message(self, message_type: MessageType, data: str = "") -> None:
+    async def send_message(
+            self,
+            message_type: MessageType,
+            data: str = ""
+    ) -> None:
         print(
-            f"Hue Light handling message of type {message_type.name} with data [{data}]."
+            f"Hue Light handling message of"
+            f" type {message_type.name} with data [{data}]."
         )
         time.sleep(TIME_TO_SLEEP)
         print("Hue Light received message.")
 
```

Comment:
In this part `time.sleep` is blocking operations and should be changed to async sleep as well.